tests/integrations/typing/test_torch_tensor.py

```diff
@@ -25,3 +27,19 @@ class MyDocument(BaseDoc):
     assert isinstance(d.embedding, TorchEmbedding)
     assert isinstance(d.embedding, torch.Tensor)
     assert (d.embedding == torch.zeros((128,))).all()
+
+
+def test_torchtensor_deepcopy():
+    # Setup
+    original_tensor_float = TorchTensor(torch.rand(10))
+    original_tensor_int = TorchTensor(torch.randint(0, 100, (10,)))
+
+    # Exercise
+    copied_tensor_float = copy.deepcopy(original_tensor_float)
+    copied_tensor_int = copy.deepcopy(original_tensor_int)
+
+    # Verify
+    assert torch.equal(original_tensor_float, copied_tensor_float)
+    assert original_tensor_float is not copied_tensor_float
+    assert torch.equal(original_tensor_int, copied_tensor_int)
+    assert original_tensor_int is not copied_tensor_int
```

Comment:
Okay let me fix that..!

Reply:
> I think this test is not enough. You should rather do
> 
> ```
> assert original_tensor_float.data_ptr() != copied_tensor_float.data_ptr()
> ```
> 
> indeed in pytorch each view of a tensor would have a different python ID but what matter is the underlying storage.
> 
> Good PR otherwise !!

Hi @samsja 
Accidently I deleted the repo from my local machine that's why I had to clone it again and regarding that I am unable to change in this merged pull request, so for to reafctor the test I need to create another PR.
So should I create a another PR regarding that?

Reply:
yes you would have to create another PR for this